R/check_forecasts.R

```diff
@@ -49,67 +49,85 @@
 #' check <- check_forecasts(example_quantile)
 #' print(check)
 #' check_forecasts(example_binary)
-
 check_forecasts <- function(data) {
```

Comment:
Add an issue outlining the potential to add a new argument specifying the target type (and potentially unit etc) the user thinks they are entering (as discussed)